kolibri/plugins/base.py

```diff
@@ -181,6 +186,31 @@ def api_url_module(self):
                 )
             return module
 
+    def root_url_module(self):
+        """
+        Return a url module, containing ``urlpatterns = [...]``, a conventional
+        Django application url module.
+
+        Do this separately for endpoints that need to be attached at the root.
+
+        URLs are by default accessed through Django's reverse lookups like
+        this::
+
+            reverse('kolibri:url_name')
+
+        By default this will be discovered based on the root_view_urls
```

Comment:
I would like a very big warning here, just to scare people.

Reply:
added warning